operational_analysis/methods/electrical_losses.py

```diff
@@ -158,7 +160,51 @@ def calculate_electrical_losses(self):
         if self._monthly_meter:
             scada_monthly = self._scada_daily.resample('MS')['corrected_energy'].sum().to_frame()
             scada_monthly.columns = ['turbine_energy_kwh']
+
+            # Determine availability for each month represented
+            scada_monthly['count'] = self._scada_sum.resample('MS')['count'].sum()
+            scada_monthly['expected_count_monthly'] = 1 #initialized values for column
+
+            #months with 31 days
+            month_list = ((scada_monthly.index.month ==1) | \
+            (scada_monthly.index.month == 3) | \
+            (scada_monthly.index.month ==5) | \
+            (scada_monthly.index.month ==7) | \
+            (scada_monthly.index.month ==8) | \
+            (scada_monthly.index.month ==10) | \
+            (scada_monthly.index.month ==12))
+            scada_monthly.loc[month_list, 'expected_count_monthly'] = \
```

Comment:
What is the "expected_count_monthly"? Could this be expressed more simply by resampling a full datetime index down to hours, aggregating by month, and then counting the rows?  Could all of this leap-year, days-in-month, hour-count stuff could be handled by datetime?